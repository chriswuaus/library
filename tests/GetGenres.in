list genres
add collection test3nodup.csv
list genres
add collection duplicates.csv
list genres
exit
