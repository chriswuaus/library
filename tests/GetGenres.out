user: No books in system.

user: 10 books successfully added.

user: Comedy
Comic
Epic
Fantasy
Historical Fiction
Horror
Poetry
War

user: 21 books successfully added.

user: Comedy
Comic
Epic
Fantasy
Historical Fiction
Horror
Poetry
War

user: Ending Library process.
